partitura/musicanalysis/performance_features.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module implement a series of mid-level descriptors of the performance expressions.
+Built upon the low-level basis functions from the performance codec. 
+"""
+
+import sys
+import types
+from typing import Union, List
+import warnings
+import numpy as np
+np.set_printoptions(suppress=True)
```

Comment:
Is that necessary?

Reply:
I think it might be related to some warnings, Huan mentioned. I don't think we should suppress numpy warnings though....

Reply:
I removed it